### PR TITLE
Unnecessary price sanity check

### PR DESCRIPTION
**Pull request by oldchili**

### CODE REVIEW

**Review by telome — APPROVED** [review]
Looks good

File: test/Deployment.t.sol [hunk 1]
```diff
@@ -39,6 +39,10 @@ interface UniV2FactoryLike {
     function createPair(address, address) external returns (address);
 }
 
+interface PoolLike {
+    function getReserves() external view returns (uint256, uint256, uint32);
```
Comment (sunbreak1211) [inline test/Deployment.t.sol:43]:
```suggestion
    function getReserves() external view returns (uint112, uint112, uint32);
```
Shouldn't be?

File: test/Deployment.t.sol [hunk 2]
```diff
@@ -99,4 +103,28 @@ contract DeploymentTest is DssTest {
         assertEq(GemLike(NST).balanceOf(UNIV2_NST_NGT_PAIR), pProxyDaiBalance);
         assertEq(GemLike(NGT).balanceOf(UNIV2_NST_NGT_PAIR), pProxyMkrBalance * 1200);
     }
+
+    function checkPriceSanityCheck(uint256 newUniPrice) public {
+        DssInstance memory dss = MCD.loadFromChainlog(LOG);
+        address pipMkr = ChainlogLike(LOG).getAddress("PIP_MKR");
+
+        vm.store(address(pipMkr), bytes32(uint256(1)), bytes32(newUniPrice));
```
Comment (sunbreak1211) [inline test/Deployment.t.sol:111]:
```suggestion
    function checkPriceSanityCheck(uint256 newPipPrice) public {
        DssInstance memory dss = MCD.loadFromChainlog(LOG);
        address pipMkr = ChainlogLike(LOG).getAddress("PIP_MKR");

        vm.store(address(pipMkr), bytes32(uint256(1)), bytes32(newPipPrice));
```

**Review by sunbreak1211 — APPROVED** [review]
Looks good

**Pull request merged**